cache.go

```diff
@@ -468,10 +468,10 @@ func (c *Cache[K, V]) Items() map[K]*Item[K, V] {
 	defer c.items.mu.RUnlock()
 
 	items := make(map[K]*Item[K, V], len(c.items.values))
```

Comment:
Just noticed this: the length parameter should probably be removed here, to avoid allocating excessive amounts of memory.

Reply:
Removed